src/components/SwapForm/Form.tsx

```
@@ -0,0 +1,220 @@
+
+    // Fee percentage
+    const FEE_PERCENTAGE = 0.001; // 0.1% fee
+
+    // Hardcoded rate (1:1 for now)
+    const EXCHANGE_RATE = 1;
+
+    const handleAmountChange = (amount: string) => {
+        setInputAmount(amount);
+
+        // If the input is empty or invalid, reset the converted and output amounts
+        if (isNaN(parseFloat(amount)) || amount === '') {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)